src/main/java/net/javadiscord/javabot/api/response/ApiResponses.java

```diff
@@ -0,0 +1,30 @@
+package net.javadiscord.javabot.api.response;
+
+/**
+ * Utility class which contains some basic JSON responses.
+ */
+public class ApiResponses {
```

Comment:
Note that Spring Web already uses Jackson and is able to serialize objects with that when using `@RestController`.

Reply:
yup, this is just for building multi-parameter json data

Reply:
This was removed